### ISSUE TITLE
Bind boolean values to accessibility attributes

### ISSUE THREAD

**Issue by JohnmarkBeaty** [vuejs/vue#7422]
### What problem does this feature solve?
As you [know](https://vuejs.org/v2/guide/syntax.html#Attributes), Vue currently removes HTML attributes bound to a boolean that evaluates to false.

Currently `<div :checked="false"></div>` renders as `<div></div>`. Perfect.

However, this does not work well for aria attributes. Per [spec](https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element), many aria attributes need to have values set to `false` for screen readers.

Ideally
 `<div :aria-expanded="false"></div>` renders to `<div aria-expanded="false"></div>`

[This fiddle]( https://jsfiddle.net/jbeaty/3htvyb2n/ ) demonstrates the desired output and my current workaround.

While `:aria-expanded="expanded ? expanded : 'false'"` works, it gets messy with multiple aria attributes. I know I could also use `strings`, but that feels even messier and more hackish to me.

### What does the proposed API look like?
The easiest way I can think of fixing this is with a `.boolean` Modifier. 

`<div :aria-expanded.boolean="falseBool"></div>` renders to `<div aria-expanded="false"></div>`

Thoughts or guidance?

**Comment by yyx990803** [comment]
If your values are always booleans, you can just do `:aria-expanded="String(expanded)"`.

**Comment by stowball** [comment]
What if `null` and `undefined` were the only values that removed the attribute entirely? Then we wouldn't have to litter our code with this "hack" everywhere (which I forget to do a lot, thus inadvertently punishing screenreader users).

Is there potential for this behaviour to change for v3?

**Comment by adambloomer** [comment]
> If your values are always booleans, you can just do `:aria-expanded="String(expanded)"`.

@yyx990803 Is this documented in the vue Docs anywhere? I have tried to search for it but can't find any mention of this. Thanks.

**Comment by Josh68** [comment]
> > If your values are always booleans, you can just do `:aria-expanded="String(expanded)"`.
> 
> @yyx990803 Is this documented in the vue Docs anywhere? I have tried to search for it but can't find any mention of this. Thanks.

I agree that this could be made explicit in the docs. It's just a matter of the binding value (which generally looks like a string) actually being an expression. Doesn't help that you can actually get away with doing `:attribute=value` or `:attribute="value"` -- it's a bit confusing (to me, at least) that this works with or without quotes.

> In the case of boolean attributes, where their mere existence implies true, v-bind works a little differently. In this example:

> `<button v-bind:disabled="isButtonDisabled">Button</button>`

> If isButtonDisabled has the value of null, undefined, or false, the disabled attribute will not even be included in the rendered <button> element.

The last sentence could add the value of `'false'`, and show the workaround of casting by doing `v-bind:aria-expanded="String(isElementExpanded)"`. This is important for attributes that should always be rendered.
